ansible/roles/test/tasks/warm-reboot-sad.yml

```diff
@@ -7,5 +7,5 @@
   include: advanced-reboot.yml
   vars:
       reboot_type: warm-reboot
-      preboot_list: ['neigh_bgp_down', 'dut_bgp_down', 'dut_lag_down', 'neigh_lag_down']
+      preboot_list: ['neigh_bgp_down:2', 'dut_bgp_down', 'dut_lag_down:2', 'neigh_lag_down:3']
```

Comment:
Why was single failure test got removed?

I think it might make more sense to pass preboot_list in from test case instead of define here. The benefit is:

1. The test case can define how many failures to inject freely.
2. Each test will pass/fail by itself. e.g. we might be able to pass single lag down, but having trouble with 2 lags down. With 2 different tests. It is easier to see which one failed.

Reply:
retained the original yml and moved the n member case to a separate one